shared/fixes/bash/auditd_data_retention_space_left.sh

```diff
@@ -2,7 +2,7 @@
 . /usr/share/scap-security-guide/remediation_functions
 populate var_auditd_space_left
 
-grep -q ^space_left /etc/audit/auditd.conf && \
+grep -q "^space_left[[:space:]]*=" /etc/audit/auditd.conf && \
```

Comment:
We can use `\s` in grep, do we want to do it, or do we want to rather go with `[[:space:]]`? I feel that it is a good idea to decide for one and declare its preference in the bash remediation guideline.

Reply:
Good point.

I'd rather stick with the POSIX standard, witch means using `[[:space:]]`.
Then, expressions as in https://github.com/OpenSCAP/scap-security-guide/blob/master/shared/fixes/bash/ensure_logrotate_activated.sh#L13 are more readable.

Also, from a quick `grep` in `shared/fixes/bash` it looks like `[[:space:]]` is used more than `\s`.